src/main.js

```diff
@@ -159,6 +159,8 @@
             if (!isNaN(date)) {
                 // #72: visible value must adjust timezone offset
                 label.set("datetime", new Date(date.getUTCFullYear(), date.getUTCMonth(), date.getUTCDate()).toISOString());
+            } else if(propName === "defaultValue"){
```

Comment:
Nice catch, though I think this check is redundant here. Also please use double quotes at line 163